website_recaptcha_v2_form/controllers/main.py

```diff
@@ -0,0 +1,85 @@
+import logging
+
+from odoo import _, http
+from odoo.exceptions import AccessDenied
+from odoo.http import request
+
+from odoo.addons.auth_signup.controllers.main import AuthSignupHome
+from odoo.addons.web.controllers.home import SIGN_UP_REQUEST_PARAMS, Home
+
+logger = logging.getLogger(__name__)
+
+SIGN_UP_REQUEST_PARAMS.add("g-recaptcha-response")
+
+
+class BinhexHome(Home):
```

Comment:
Don't use company name in classes, functions nor variables, it should only appear in the manifest as author and the contributors readme.
```suggestion
class RecaptchaHome(Home):
```

Reply:
ok @Christian-RB